### PR TITLE
update go.mod to add `queue.spec.Affinity`

### PR DESCRIPTION
**Pull request by qiankunli**
I want to support nodegroup in volcano, related pr https://github.com/volcano-sh/volcano/pull/2224, which needs to add `queue.spec. Affinity` field on Queue first, related pr https://github.com/volcano-sh/apis/pull/72
operate history
1.  change version of `volcano.sh/apis` from ` v1.5.0-beta.0.0.20220427084719-22d3dd007a8f` to `v0.0.0-20220705062437-edd428c7d2fd`   in go.mod, edd428c7d2fd is latest commit of `volcano.sh/apis` 
2. go mod tidy
3. go mod vendor
4. hack/update-gencode.sh
5. make update-development-yaml

I will continue to push https://github.com/volcano-sh/volcano/pull/2224 after the pr is merged.

### CODE REVIEW

**Comment by hwdef** [comment]
There is a lot of jobflow code in this pr,I don't recommend merging if there is no urgent need to merge.We are preparing to submit the code of jobflow, which may conflict with our code,

File: go.mod [hunk 1]
```diff
@@ -29,9 +29,9 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.23.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
 	stathat.com/c/consistent v1.0.0
-	volcano.sh/apis v1.5.0-beta.0.0.20220427084719-22d3dd007a8f
+	volcano.sh/apis v0.0.0-20220630011627-e579d851e770
```
Comment (hwdef) [inline go.mod:34]: The apis has been updated, please synchronize the latest commit number

**Comment by qiankunli** [comment]
> @qiankunli Please execute `go mod vendor` to update the dependencies locally and push again.

updated

File: go.mod (hunk 1 above)
Comment (qiankunli) [reply go.mod]: updated

**Comment by william-wang** [comment]
@qiankunli Looks good to me.  Please track all releated pr under issue #1830.
It's a good habit to add a subtask and a conresponding pr with it :)

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**